vcpkg/get_started/get-started-packaging.md

```diff
@@ -0,0 +1,318 @@
+---
+title: Create and publish packages
+description: Tutorial guides the user through the process of packaging a library for vcpkg.
+zone_pivot_group_filename: zone-pivot-groups.json
+zone_pivot_groups: operating-systems
+author: JavierMatosD
+ms.author: javiermat
+ms.topic: tutorial
+ms.date: 08/23/2023
+ms.prod: vcpkg
+#CustomerIntent: As a beginner C++ developer, I want to learn how to package libraries for vcpkg.
+---
+
+# Tutorial: Create and publish packages
+
+This tutorial shows you how to publish a simle library for the vcpkg registry.
+
+## Prerequisites
+
+- A terminal
+- A C++ compiler
+- [CMake](https://cmake.org/download/)
+- [Git](https://git-scm.com/downloads)
+
+::: zone pivot="os-windows"
+
+> [!NOTE]
+> For Windows users, Visual Studio's MSVC (Microsoft Visual C++ Compiler) is the required compiler for C++ development.
+
+::: zone-end
+
+## 1 - Set up vcpkg
+
+[!INCLUDE [setup-vcpkg](includes/setup-vcpkg.md)]
+
+## 2 - Configure the `VCPKG_ROOT` environment variable
+
+To set the `VCPKG_ROOT` environment variables, run the following commands:
+
+::: zone pivot="os-linux,os-macos"
+
+```bash
+export VCPKG_ROOT=/path/to/vcpkg
+export PATH=$VCPKG_ROOT:$PATH
+```
+
+> [!NOTE]
+> Setting the `VCPKG_ROOT` environment variable using the `export` command only affects the current shell session. To make this change permanent across sessions, you'll need to add the `export` command to your shell's profile script (e.g., `~/.bashrc` or `~/.zshrc`).
+
+::: zone-end
+
+::: zone pivot="os-windows"
+
+```console
+set VCPKG_ROOT="C:\path\to\vcpkg"
+set PATH=%VCPKG_ROOT%;%PATH%
+```
+
+> [!NOTE]
+> Setting the `VCPKG_ROOT` environment variable using the `set` command only affects the current shell session. To make this change permanent across sessions, you can use the `setx` command and restart the shell session.
+
+::: zone-end
+
+Setting `VCPKG_ROOT` tells vcpkg where your vcpkg instance is located.
+Adding it to `PATH` ensures you can run vcpkg commands directly from the shell.
+
+## 3 - Set up the port
+
+To create the `portfile.cmake` and `vcpkg.json` files for the new port, run:
+
+```console
```

Comment:
We should separate the command-line from its output so that users can click on the "Copy to clipboard" button and paste the command on their terminal